nominees/provenance-services-with-smart-data-agreements-using-data-exchange-protocols.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Provenance Services with Smart Data Agreements using Data Exchange Protocols",
+  "aliases": [
+    "DEXA",
+    "PS-SDA"
+  ],
+  "description": "DEXA protocol is a key component for developing and deploying trustworthy, human-centric decentralised data applications (dApps) across domains. ",
+  "website": "https://ontochain.ngi.eu/content/ps-sda",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/decentralised-dataexchange/data-exchange-agreements/blob/main/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 9,
+      "evidenceText": "The DEXA protocol facilitates sustainable personal exchange for organisations, ensuring that all data transactions are lawful, scalable and verifiable. An organisation can be a Data Source or a Data Using Service. The DEXA protocol brings in the requisite trust and governance to establish a ubiquitous data exchange marketplace while empowering individuals to be in control of their data. All organisations need to ensure that they are on the right side of the law (e.g. the GDPR) when consuming personal data (risk management) and to establish the digital trust needed for individuals to say yes to sharing their data. "
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
```

Comment:
I think this project falls more under the category of open standard rather than open software. As I understand the main component of the project is the [PS-SDA protocol](https://github.com/decentralised-dataexchange/data-exchange-agreements) and has some complementary software pieces like the [DEXA SDK](https://github.com/decentralised-dataexchange/dexa-sdk).

Is this right? @lalc

Reply:
Its primarily open-source software (DEXA SDKs) implements the protocol which we are proposing to contribute as a DPG

Reply:
Ok thanks for the clarification.